app/frontend/wallet/shelley/shelley-ledger-crypto-provider.ts

```diff
@@ -211,7 +210,7 @@ const ShelleyLedgerCryptoProvider = async ({
     return Object.entries(tokenObject).map(([policyId, assets]) => {
       const tokens = assets.map(({assetName, quantity}) => ({
         assetNameHex: assetName,
-        amountStr: quantity.toString(),
+        amount: quantity.toString(),
```

Comment:
what type is quantity? We support numbers (if they are less than max safe integer) and native bigints. Are we using some non-native bigint library? :thinking:

Reply:
quantity is of type `number` as everything currently in the project. I believe there is already an issue for migrating to native bigint.

Reply:
yes, but the point is the current ledgerjs api accepts all three representations of numbers - BigInt, number (up to max safe integer) or stringified number

Reply:
Yes I am aware I see that probably Pershing point and also yours was that why we are converting it to string if it accepts numbers from now on, the answer is that I ve just chosen the most straightforward way of integrating the ledger app and haven't given this a thought. Either way, I ll remove the conversion to string. Thnx.